projects/m3/003-roman-numerals/solutions/pietrorosano/solution_003-roman-numerals.py

```diff
@@ -0,0 +1,32 @@
+# Create a recursive function that converts a Roman numeral to an integer.
+
+# Your function should process one or two characters at the beginning of the string,
+# and then call itself recursively on all of the unprocessed characters.
+# Use an empty string, which has the value 0, for the base case.
+
+# In addition, write a main program that reads a Roman numeral from the user and displays its value.
+
+print("\nEnter a roman number")
+inp = input()
+
+def roman_num_to_int(rom, count=0):
+    romans_list = list(rom)
+    roman_numbers = {"M":1000, "D":500, "C":100, "L":50, "X":10, "V":5, "I":1}
```

Comment:
When you have to define values that doesn't change, a better approach is to put them outside functions, in order to not assign them recursively